emission/core/wrapper/suggestion_sys.py

```diff
@@ -211,13 +231,16 @@ def find_destination_business(lat, lon):
             % str(return_tuple))
         return return_tuple
 
+
 ### BEGIN: Pulled out candidate functions so that we can evaluate individual accuracies
 def category_of_business_awesome(lat, lon):
     return []
 
-def category_from_name(business_name):
+def category_from_name_new(business_id, business_cat_json):
```

Comment:
Please do not rename and modify existing functions until we have compared yelp to the alternatives.